main.tex

```diff
@@ -309,7 +309,7 @@ \subsection{Measuring Extortion from the History of Play}
 
 We can measure how close a strategy is to being zero determinant using standard
 linear algebraic approaches. Essentially we attempt to find \(x=(\alpha,
-\beta)\) and \(p^*=(\tilde p_1 - 1, \tilde p_2 - 1, \tilde p_3, \tilde p_4)\)
+\beta)\) and \(p^*=(p_1 - 1, p_2 - 1, p_3, p_4)\)
```

Comment:
This is the same as saying p^*=\tilde p, no? Maybe just say we try to find the distance to the closest ZD strategy to p, that is \to find \alpha and \beta that is the best solution Cx = \tilde p....

Reply:
Yup :+1: thanks @marcharper will push that now.